gallery/src/demos/demo-ha-config-navigation.ts

```diff
@@ -0,0 +1,80 @@
+import { mdiDevices, mdiHomeAssistant, mdiRobot, mdiRocket } from "@mdi/js";
+import { css, html, LitElement, TemplateResult } from "lit";
+import { customElement, property } from "lit/decorators";
+import "../../../src/components/ha-card";
+import { provideHass } from "../../../src/fake_data/provide_hass";
+import { PageNavigation } from "../../../src/layouts/hass-tabs-subpage";
+import "../../../src/panels/config/dashboard/ha-config-navigation";
+import { HomeAssistant } from "../../../src/types";
+
+const PAGES: PageNavigation[] = [
+  {
+    path: "#demo-ha-config-navigation",
+    name: "Devices & Services",
+    description: "Manage integrations, areas, tags and helpers",
+    iconPath: mdiDevices,
+    iconColor: "#004E98",
+  },
+  {
+    path: "#demo-ha-config-navigation",
+    name: "Automations",
+    description: "Manage integrations, areas, tags and helpers",
+    iconPath: mdiRobot,
+    iconColor: "#2A850E",
+  },
+  {
+    path: "#demo-ha-config-navigation",
+    name: "Add-ons & Backups",
+    description: "Create backups, check logs or reboot your system",
+    iconPath: mdiHomeAssistant,
+    iconColor: "#44739E",
+  },
+  {
+    path: "#demo-ha-config-navigation",
+    name: "Basic icon with no background",
+    description:
+      "This is just here to display that the component can handle this to",
+    iconPath: mdiRocket,
+  },
+];
+
+@customElement("demo-ha-config-navigation")
+export class DemoHaConfigNavigation extends LitElement {
```

Comment:
I'm a bit 🤷 on this gallery page. Like… it's used on one page and requires no set-up to see how it renders things. It's not a component that we reuse.